sound/soc/codecs/rt715-sdca-sdw.c

```diff
@@ -234,10 +234,10 @@ static int __maybe_unused rt715_dev_resume(struct device *dev)
 	if (!slave->unattach_request)
 		goto regmap_sync;
 
-	time = wait_for_completion_timeout(&slave->enumeration_complete,
+	time = wait_for_completion_timeout(&slave->initialization_complete,
 					   msecs_to_jiffies(RT715_PROBE_TIMEOUT));
 	if (!time) {
-		dev_err(&slave->dev, "%s: Enumeration not complete, timed out\n", __func__);
+		dev_err(&slave->dev, "%s: Initialization not complete, timed out\n", __func__);
```

Comment:
commit message 
s/perpheral/peripheral

Also the fixes tag is not completely correct, the confusion existed before f892e66, i just changed the error string in that commit.

Reply:
> commit message s/perpheral/peripheral

Updated.

> 
> Also the fixes tag is not completely correct, the confusion existed before [f892e66](https://github.com/thesofproject/linux/commit/f892e66fcabc6161cd38c0fc86e769208174b840), i just changed the error string in that commit.

Yes, the issue exists since 20d17057f0a8c, but the commit can only apply to https://github.com/thesofproject/linux/commit/f892e66fcabc6161cd38c0fc86e769208174b840

Reply:
I don't see any update, still the same typo in the commit message?

Also it's best to capture that "the issue exists since 20d1705, but the commit can only apply to f892e66" for people doing backports for linux-stable.